### DISCUSSION TITLE
[Enhancement] [table-v2] Table-V2 是否可以提供基本的栏位宽度拖拉改变及边框功能

### DISCUSSION THREAD

**Discussion by lrthorhammer007** [element-plus/element-plus#13146] [Feature Request]
Feature type: **`Enhancement`**

## Details

- Enhanced component name: `table-v2`

## Description

Table-V2没有栏位宽度拖拉改变功能，cell中文本内容较多 无法显示。希望能够支持栏位宽度改变功能

### Additional comments

_(empty)_

**Comment by Muspcn** [comment]
确实很需要这个功能
